src/style/sections/libraries/libraries-section.less

```diff
@@ -21,86 +21,268 @@
  *
  */
  
- @graphic-drag-preview-size: 64px;
+@graphic-drag-preview-size: 64px;
+@libraries-button-size: 1.1rem;
+@libraries-preview-height: 40px;
+@libraries-preview-width: 60px;
+
 
 .libraries {
     position: relative;
     min-height: 7.5rem;
     flex-grow: 1;
     flex-shrink: 0;
-}
-
-.libraries__content {
-}
-
-.libraries__drop-overlay {
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    right: 0;
-    left: 0;
-    background-color: rgba(0,0,0,0.2);
-    z-index: 1;
-    cursor: copy;
-}
-
-.libraries__drop-overlay_disallow {
-    cursor: not-allowed;
-}
-
-.libraries .section-container {
-    flex-basis: 17.5rem;
-}
-
-.libraries .section-container__collapsed {
-    display: none;
-}
-
-// .libraries.section.section__sibling-collapsed {
-//     flex-grow: 999999;
-// }
-
-.library-color-swatch {
-    width: 50px;
-    height: 40px;
-    display: flex;
-    align-items: center;
-    flex-grow: 0;
-    flex-shrink: 0;
-    box-sizing:  border-box;
-    padding: 0;
-    margin: 0;
-    justify-content: center;
-    overflow: hidden;
-}
-
-.libraries__split-button-list{
-    display: none;
-}
-
-.libraries .split-button__item svg {
-    width: 1.4rem;
-    height: 1.4rem;
-}
-
-.split-button__item__color-icon {
-    width: 1.2rem;
-    height: 1.2rem;
-    border: 1px solid white;    
-    border-radius: 1px;
-}
-
-.libraries-bar{
-    ul {
-        flex-grow: 0;
-        .split-button__item {
-            width: 30px;
+    
+    
+    .section-title {
+        border-bottom: @hairline solid @layer-divider;
+    }
+    
+    .section-container {
+        padding: 0;
+    }
+    
+    .split-button__item {
+        svg {
+            width: @libraries-button-size;
+            height: @libraries-button-size;
+        }
+    }
+    
+    
+    .section-container {
+        flex-basis: 17.5rem;
+        
+        &__collapsed {
+            display: none;
+        }
+    }
+    
+    .assets__graphic__dragging {
+        opacity: 0.7;
+    }
+    
+    .libraries-bar{
+        ul {
+            flex-grow: 0;        
+            .split-button__item__color-icon {
+                width: @libraries-button-size;
+                height: @libraries-button-size;
+                border: 1px solid white;    
+                border-radius: 1px;
```

Comment:
Actually, @designedbyscience: since this is such an important new body of CSS, can you do a quick review of it yourself?